### PR TITLE
feat: Control return to confirm action

### LINKED ISSUE #10213: Control return to confirm action
We should introduce something like that for modals 

apply this action for all confirmation modals

command+return/control+return

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

continue #10306

@Jarsen136 let me know if you find anything, this approach seems to fix the issues that had previously and it's simpler imo

- [x] Closes #10213

## Needs Design check

- @exezbcz please review

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2024-10-17 at 12 22 44

### CODE REVIEW

File: composables/useKeyboardKeys.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  return watch([Meta, Control, Enter], () => {
+    if ((Meta.value || Control.value) && Enter.value) {
+      callback()
+    }
+  })
+}
+
+export function useKeyboardKeys({ onPressControlEnter: handlePressControlEnter }) {
+  handlePressControlEnter && onPressControlEnter(handlePressControlEnter)
+}
```
Comment (Jarsen136) [inline composables/useKeyboardKeys.ts:15]:
```suggestion
function handlePressControlEnter(callback: () => void) {
  const { Meta, Control, Enter } = useMagicKeys()

  return watch([Meta, Control, Enter], () => {
    if ((Meta.value || Control.value) && Enter.value) {
      callback()
    }
  })
}

export function useKeyboardKeys({ onPressControlEnter }) {
  onPressControlEnter && handlePressControlEnter(onPressControlEnter)
}
```

**Review by Jarsen136 — COMMENTED** [review]
otherwise looks ok to me

File: composables/useKeyboardKeys.ts (hunk 1 above)
Comment (hassnian) [reply composables/useKeyboardKeys.ts:15]: I'd rather have `onPressControlEnter` as a helper method like `onModalAnimation`

`useKeyboardKeys` is not what I'm using

Comment (hassnian) [reply composables/useKeyboardKeys.ts]: done f8b92a7


CleanShot 2024-10-19 at 07 07 12

**Review by Jarsen136 — APPROVED** [review]

**Pull request merged**